src/mozillavpn.cpp

```diff
@@ -2080,6 +2073,14 @@ void MozillaVPN::registerInspectorCommands() {
         return QJsonObject();
       });
 
+  InspectorHandler::registerCommand(
+      "mockSomethingWentWrong",
+      "Force the UI to show the Something Went Wrong screen", 0,
```

Comment:
Is it better to use the following, because it is triggering a backend failure:
      "force_backend_failure", "Force a backend failure", 0,